addons/mail/models/mail_message.py

```diff
@@ -85,6 +85,7 @@ def default_get(self, fields):
     # content
     subject = fields.Char('Subject')
     date = fields.Datetime('Date', default=fields.Datetime.now)
+    edit_date = fields.Datetime('last edit DateTime')
```

Comment:
As we globally never update mesages, and as editing a message is something that is not done often, I would not define yet another field on this heavily used table, as there is already "write_date" that is available. As we have millions of message, this would require an upgrade and DB space for few added value compared to using 'write_date'. Did you find any flow that writes on the message and would lead to an invalid "write_date" ?